helm/prometheus-agent/charts/prometheus-agent/values.yaml

```
@@ -86,6 +86,31 @@ serviceMonitorSelector: {}
 podMonitorNamespaceSelector: {}
 podMonitorSelector: {}
 
+# scrapeClasses allows to define relabelings for specific scrape classes that can be used in ServiceMonitors and PodMonitors.
+# More info: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.ScrapeClass
+scrapeClasses:
+  - name: giantswarm-servicemonitors
```

Should this be giantswarm or giantswarm-services? Because this is also for podmonitors right?
```suggestion
  - name: giantswarm-services
```

---

I'm not sure I understand why we need 2?

---

Because I checked the kyverno policies and PodMonitor has one more relabeling than ServiceMonitor:
https://github.com/giantswarm/kyverno-policies-observability/blob/main/policies/observability/PodMonitor.yaml

---

I think we can have the 2 in one scrape class though. WDYT?

---

If we make it default, then this goes against the plug and play approach :)

---

In that case we don't need it in the pod monitor as well

---

I mean, the node is interesting as a label and I think it's there in servicemonitors by default